egs2/ipapack/asr1/conf/tuning/train_asr_transformer.yaml

```diff
@@ -0,0 +1,52 @@
+seed: 11711
+use_amp: true
+num_workers: 6
+log_interval: 50  # debugging only
+batch_type: numel
+num_iters_per_epoch: 5000
+batch_bins: 7000000
+max_epoch: 100
+patience: none
+# The initialization method for model parameters
+init: xavier_uniform
+best_model_criterion:
+-   - valid
+    - acc
+    - max
+keep_nbest_models: 10
+
+encoder: transformer
+encoder_conf:
+    output_size: 512
+    attention_heads: 8
+    linear_units: 2048
+    num_blocks: 12
+    dropout_rate: 0.1
+    positional_dropout_rate: 0.1
+    attention_dropout_rate: 0.0
+    input_layer: conv2d
+    # 40 ms resolution (1/4 length)
```

Comment:
this could actually be hurting performance if using XEUS features

since the frame rate of XEUS is 50 fps

I will use conv2d1 (no downsampling) in the future

Reply:
Usually for ASR, we use conv2d2 for SSL feature. (50% downsample)

Using half is a good balance to efficiency and modeling performance. But maybe phone would be a different story?